tests/test_brozzling.py

```diff
@@ -72,6 +72,16 @@ def do_GET(self):
             else:
                 super().do_GET()
 
+        def do_POST(self):
+            if self.path == '/login-action':
+                self.send_response(200)
+                payload = b'login successfull\n'
+                self.send_header('Content-Type', 'text/plain;charset=utf-8')
+                self.send_header('Content-Length', len(payload))
+                self.end_headers()
+                self.wfile.write(payload)
+
```

Comment:
any reason not to add this?
```
else:
    super().do_POST
```

Reply:
I just missed it. Thank you! I've just added it.